src/databricks/labs/ucx/mixins/fixtures.py

```diff
@@ -638,6 +639,47 @@ def _scim_values(ids: list[str]) -> list[iam.ComplexValue]:
     return [iam.ComplexValue(value=x) for x in ids]
 
 
+def _wait_group_provisioned(interface: AccountGroupsAPI | GroupsAPI, group: Group) -> None:
```

Comment:
please do it into https://github.com/databrickslabs/pytester

Reply:
Yes, I'll move these changes over there.

Reply:
do we need this PR here then?

Reply:
Nope; it can be closed.